util/module_logger_test.go

```diff
@@ -0,0 +1,253 @@
+package util_test
+
+import (
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/suite"
+	"github.com/uber-common/bark"
+	"github.com/uber/ringpop-go/test/mocks"
+	"github.com/uber/ringpop-go/util"
+	"testing"
+)
+
+type MockLogger struct {
+	*mocks.Logger
+}
+
+func (ml *MockLogger) On(methodName string, arguments ...interface{}) *mock.Call {
+	return ml.Logger.On(methodName, arguments)
+}
+
+func (ml *MockLogger) Onf(methodName string, format string, arguments ...interface{}) *mock.Call {
+	return ml.Logger.On(methodName, format, arguments)
+}
+
+func newMockLogger() *MockLogger {
+	return &MockLogger{Logger: &mocks.Logger{}}
+}
+
+type ModuleLoggerSuite struct {
+	suite.Suite
+	ml *util.ModuleLogger
```

Comment:
Note that the `ml` name for "module logger" got me confused with the mock logger when reading the tests. Maybe `ml` should just be `logger` or `baseLogger` or something.